test/compilable/interpret3.d

```diff
@@ -897,13 +897,6 @@ int staticdynamic()
 }
 static assert(staticdynamic() == 0);
 
-int[] crashing()
-{
-    int[12] cra;
-    return (cra[2 .. $] = 3);
-}
-static assert(crashing()[9] == 3);
```

Comment:
Why did you remove that test? It's surely escaping something, but the test has nothing to do with `-transition=safe/-dip1000`. It's also not `@safe` code.